### ISSUE TITLE
Add fuzzy to all and any

### ISSUE THREAD

**Issue by wroughtec** [base33/umbraco.examine.linq#4]
Is it possible to add fuzzy logic to ContainsAll and Contains Any? As currently I gte the following error if I add:
````
Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code.

Exception Details: Lucene.Net.QueryParsers.ParseException: Encountered " ")" ") "" at line 1, column 141.
Was expecting one of:
   <EOF>
   <AND> ...
   <OR> ...
   <NOT> ...
   "+" ...
   "-" ...
   "(" ...
   "*" ...
   "^" ...
   <QUOTED> ...
   <TERM> ...
   <PREFIXTERM> ...
   <WILDTERM> ...
   "[" ...
   "{" ...
   <NUMBER> ...
   

Source Error: 
````

**Comment by base33** [comment]
Absolutely.  It is a feature on the way very soon.  Along with getting all of type.

e.g.
var products = new Index<Product>().ToList();

[NodeTypeAlias("Product")]
public class Product
{
  //propertys
}

**Comment by wroughtec** [comment]
Cool look forward to it.

Also I found that adding Contains(term, fuzzy) gave an error while Contains(term).Fuzzy(0.5) did not?

**Comment by base33** [comment]
Hey Will,

I will be looking into this bug and adding this functionality too.

Should be in v1.2.

Thanks 

Craig

**Comment by base33** [comment]
Hi Will,

This is now available in the latest version 1.2.

You can now do:

Index<SomeDocType>().Where(r => r.Name.ContainsAll("x1", "x2", "x3").Fuzzy(0.7)).ToList();

Fuzzy is now just through the extension method.  So is boost.

Cheers,

Craig
